core/variant.cpp

```diff
@@ -1589,14 +1617,12 @@ String Variant::stringify(List<const void *> &stack) const {
 		case OBJECT: {
 
 			if (_get_obj().obj) {
-#ifdef DEBUG_ENABLED
-				if (ScriptDebugger::get_singleton() && _get_obj().ref.is_null()) {
+
+				if (ScriptDebugger::get_singleton() && !_get_obj().id.is_reference() && ObjectDB::get_instance(_get_obj().id) == nullptr) {
 					//only if debugging!
```

Comment:
Since the other _only if debugging_ comments have gone away, this one should too.

Please don't kill me for this.

Reply:
alright